BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Extensibility/Implementation/ExceptionConverterTest.cs

```diff
@@ -185,6 +185,19 @@ public void DoesNotTrimShortExceptionMessages()
             Assert.AreEqual(5, expDetails.message.Length);
         }
 
+        [TestMethod]
+        [DataRow(null)]
+        [DataRow("")]
+        [DataRow(" ")]
+        public void ProvidesDefaultMessage(string message)
```

Comment:
What's the intention of this parameter? I don't see it used.

Reply:
my comment is a dupe of https://github.com/microsoft/ApplicationInsights-dotnet/pull/2857#discussion_r1526645511